docs/root/version_history/current.rst

```diff
@@ -28,6 +28,7 @@ Bug Fixes
 * csrf: fixed issues with regards to origin and host header parsing.
 * dynamic_forward_proxy: only perform DNS lookups for routes to Dynamic Forward Proxy clusters since other cluster types handle DNS lookup themselves.
 * fault: fixed an issue with `active_faults` gauge not being decremented for when abort faults were injected.
+* grpc-web: fixed an issue with failing HTTP/2 requests on some browsers. Notably, WebKit-based browsers (https://bugs.webkit.org/show_bug.cgi?id=210108), Internet Explorer 11, and Edge (pre-Chromium).
```

Comment:
Release note should go back to being general, and include the runtime guard

Reply:
This is a side-effect of skipping encoding empty trailers, could we keep this one as well?

Reply:
Yeah this is fine, thanks.